crio/manifest.json

```diff
@@ -0,0 +1,23 @@
+{
+  "categories": [
+    "containers"
+  ],
+  "creates_events": false,
+  "display_name": "CRI-O",
+  "guid": "6eb96c6a-3e2d-4236-9387-fa3b0c455336",
+  "is_public": false,
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "metric_prefix": "crio.",
+  "metric_to_check": "crio.operations.count",
+  "name": "crio",
+  "public_title": "Datadog-Crio Integration",
+  "short_description": "Track all your CRI-O metrics with Datadog",
+  "support": "core",
+  "supported_os": [
```

Comment:
We should list `linux` only

Reply:
Actually, I was hesitating with this as the check can run on every platform but not CRI-O itself so you could potentially have a agent running on a mac that's running the check over the network where CRI-O is running? WDYT?

Reply:
seems that's what we're doing for `kube_dns` / `kube_proxy`, lgtm